pkgs/by-name/sq/sqlite-vec/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  gettext,
+  fetchFromGitHub,
+  sqlite,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sqlite-vec";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "asg017";
+    repo = "sqlite-vec";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-Baqe/1DxZsHtYjuwGJD5hI0GFlpxxaQDsx8xDBthHrI=";
+  };
+
+  nativeBuildInputs = [ gettext ];
+
+  buildInputs = [ sqlite ];
+
+  makeFlags = [
+    "loadable"
+    "static"
+  ];
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm444 -t "$out/lib" \
+      "dist/libsqlite_vec0${stdenv.hostPlatform.extensions.staticLibrary}" \
+      "dist/vec0${stdenv.hostPlatform.extensions.sharedLibrary}"
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "vector search SQLite extension that runs anywhere";
```

According to the [recommendations for meta attributes](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes) the description should start with a capital letter:
```suggestion
    description = "Vector search SQLite extension that runs anywhere";
```